scripts/copy_run_recipe.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import re
+import os
+import sys
+import json
+
+from argparse import ArgumentParser
+from datetime import datetime
+from genologics.lims import Lims
+from genologics.entities import Process
+from genologics.config import BASEURI, USERNAME, PASSWORD
+
+DESC = """EPP for copying run recipe"""
+
+def main(lims, args):
+    log=[]
+    content = None
```

Comment:
You can remove these `log` (as we are actually doing anything with it) and `content` (as it defined in line 26)